system/models/users/user.py

```diff
@@ -9,21 +10,21 @@ def __init__(self, user_id: int, user_name: str, password: str, is_admin: bool):
         self._password = password
         self._is_admin = is_admin
 
-    def get_user(self):
-        return [self._user_id, self._user_name, self._password, self._is_admin]
-
-    def get_user_id(self):
+    def get_user_id(self) -> int:
         return self._user_id
 
-    def get_user_name(self):
+    def get_user_name(self) -> str:
         return self._user_name
 
-    def get_password(self):
+    def get_password(self) -> str:
         return self._password
 
-    def get_is_admin(self):
+    def get_is_admin(self) -> bool:
         return self._is_admin
 
+    def get_user_as_list(self) -> List[str]:
+        return [str(self._user_id), self._user_name, self._password, str(self._is_admin)]
+
```

Comment:
The file looks good, just this function "get_user_as_list" I didn't understand why we need to return a list of all entry